datacube_stats/__init__.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import absolute_import
```

Comment:
Does the code have to be compatible forward to 2.5?

Reply:
No. I don't know why `pylint` insists on this.

Do you think it is a good idea just to disable absolute import and delete all these superfluous declarations? Will do it then.

Reply:
I think pylint sees `from __future__ ...` and assumes that py2 is needed, and then complains about other missing future imports, try removing `from __future__` imports altogether and see if it still complains.